renderdoc/os/win32/win32_stringio.cpp

```diff
@@ -186,6 +186,48 @@ string GetFullPathname(const string &filename)
   return StringFormat::Wide2UTF8(wstring(path));
 }
 
+string FindFileInPath(const string &file)
+{
+  string filePath;
+
+  // Search the PATH directory list for the application (like shell where) to get the absolute path
+  // Return "" if no exectuable found in the PATH list
+  const DWORD size = 65535;
+  char envPath[size];
+  if(GetEnvironmentVariableA("PATH", envPath, size) == 0)
+    return filePath;
+
+  char *next = nullptr;
+  const char *pathSeparator = ":";
```

Comment:
Strange, that was a semicolon at one point.  I think I experimented with putting the string directly in the parameter list, then accidentally switched it when reverting.  Will fix, for sure.